src/test/java/com/zombiecastlerush/building/InventoryTest.java

```diff
@@ -0,0 +1,36 @@
+package com.zombiecastlerush.building;
+
+import com.zombiecastlerush.role.Player;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class InventoryTest {
+    private Inventory inventory = new Inventory();
+    @Before
+    public void setUp() throws Exception {
+        inventory.addItems(new Item("item0","item0"));
+        inventory.addItems(new Item("item1","item1"));
+        inventory.addItems(new Item("item2","item2"));
+        inventory.addItems(new Item("item3","item3"));
+    }
+
+    @Test
+    public void getItemsReturnsListOfCurrentItemsForRequestingRole() {
+        List<Item> itemsList = inventory.getItems();
+        System.out.println(itemsList);;
```

Comment:
I assume you meant to write an assert here?

Reply:
Yes, I will continue to work on the unit test to use assert to check the rest of the methods.

Reply:
@kaashis , @gtulkn12 , @wang-xiaotian , should we gate PRs on unit tests?

Reply:
@gtulkn12 

- [ ] Write unit tests for addItems and deleteItems()

Reply:
Unit Test Complete